### ISSUE TITLE
Cobertura report problem with filename clash

### ISSUE THREAD

**Issue by alclonky** [jenkinsci/code-coverage-api-plugin#80]
If 2 packages have files with the same filename, the report is broken. All classes with same filename of package test2 are in package test1 after the transformation!

Example Input
```
<?xml version="1.0"?>
<!DOCTYPE coverage SYSTEM "http://cobertura.sourceforge.net/xml/coverage-04.dtd">

<coverage line-rate="0.308048871212338" branch-rate="0.27936393205637877" lines-covered="10766" lines-valid="34949" branches-covered="4638" branches-valid="16602" complexity="0.0" version="2.1.1" timestamp="1556499335156">
	<sources>
	</sources>
	<packages>
		<package name="test1" line-rate="0.0" branch-rate="0.0" complexity="0.0">
			<classes>
				<class name="test1.test1" filename="test.java" line-rate="0.0" branch-rate="0.0" complexity="0.0">
					<methods>
						<method name="main" signature="([Ljava/lang/String;)V" line-rate="0.0" branch-rate="0.0" complexity="0">
							<lines>
								<line number="57" hits="0" branch="false"/>
								<line number="70" hits="0" branch="false"/>
								<line number="74" hits="0" branch="false"/>
								<line number="76" hits="0" branch="false"/>
							</lines>
						</method>
					</methods>
				</class>
			</classes>
		</package>
		<package name="test2" line-rate="0.0" branch-rate="0.0" complexity="0.0">
			<classes>
				<class name="test2.test2" filename="test.java" line-rate="0.0" branch-rate="0.0" complexity="0.0">
					<methods>
						<method name="log" signature="(Ljava/lang/String;)V" line-rate="0.0" branch-rate="1.0" complexity="0">
							<lines>
								<line number="97" hits="0" branch="false"/>
								<line number="98" hits="0" branch="false"/>
								<line number="100" hits="0" branch="false"/>
							</lines>
						</method>
					</methods>
				</class>
			</classes>
		</package>
	</packages>
</coverage>

```
Output after cobertura-to-standard.xsl transformation:
```
<?xml version="1.0"?>
<report name="cobertura">
  <group>
    <package name="test1">
      <file name="test.java">
        <class name="test1.test1">
          <method name="main" signature="([Ljava/lang/String;)V">
            <line number="57" hits="0" branch="false"/>
            <line number="70" hits="0" branch="false"/>
            <line number="74" hits="0" branch="false"/>
            <line number="76" hits="0" branch="false"/>
          </method>
        </class>
        <class name="test2.test2">
          <method name="log" signature="(Ljava/lang/String;)V">
            <line number="97" hits="0" branch="false"/>
            <line number="98" hits="0" branch="false"/>
            <line number="100" hits="0" branch="false"/>
          </method>
        </class>
      </file>
    </package>
    <package name="test2"/>
  </group>
</report>
```

**Comment by tempora-mutantur** [comment]
Hi,
I guess you are using `coberturaAdapter` here.
Please, try Cobertura plugin version 1.15 and Code Coverage API 1.1.0. This was resolved there.
Thank you.
cc @cizezsy

**Comment by cizezsy** [comment]
This should be fixed in version 1.1.0 :)
